File: Breeze.Domain/Income.cs
namespace Breeze.Domain
{
    public class Income
    {
        public int Id { get; set; }
        public string UserId { get; set; }
        public Budget Budget { get; set; }
        public string Name { get; set; }
        public decimal Amount { get; set; }
        public int Year { get; set; }
        public int Month { get; set; }
        public int Day { get; set; }
    }
}

File: Breeze.Web/src/services/models/income.ts
export type Income = {
	id?: number
	userId?: string
	budgetId?: number
	name: string
	date: Date
	amount: number
}

// This is used to initialize a new income
export const emptyIncome: Income = {
	name: 'Income Name',
	date: new Date(),
	amount: 0,
}